Sitka-Sound-Science-Center/sitka-landslide
# Visual design for detail view

## Overview

Apply final visual design for detail view:

- Dot for current risk should be right aligned on chart with no gap
- Add back the arrow that points to the current risk dot
- Adds legend
- General style cleanup

Resolves #20

### Demo
<img width="391" alt="Screen Shot 2022-04-25 at 8 11 11 PM" src="https://user-images.githubusercontent.com/1809908/165194410-dbf1b3ad-7cd3-492c-bd36-4ebfff31bee3.png">

## Testing Instructions

 * Look at detail view with low, medium, and high risk, at different viewport sizes
 * Make sure nothing looks weird
 * I snuck in a legend on the map view too—take a look there also

## Checklist
